i18n/th_TH.csv

```diff
@@ -7,3 +7,5 @@
 "Print", "พิมพ์"
 "Choose your installment terms","เลือกระยะเวลาการผ่อนชำระ"
 "Miminum order value is 3000 THB", "ยอดสั่งซื้อขั้นต่ำ 3000 บาท"
+"Choose number of monthly payments", "เลือกระยะเวลาผ่อนชำระ"
+"%terms months (%amount THB / month)", "%terms เดือน (%amount บาท/เดือน)"
```

Comment:
Should we be using Magento's formatting function for a currency amount?

Reply:
we don't use magento function for for formatting.

Reply:
i got that from framework.. followed what they did in magento.

Reply:
Fair enough. It's handled by magento functions in Mag1